addons/account/models/account_move.py

```diff
@@ -102,8 +102,7 @@ def _onchange_date(self):
     partner_id = fields.Many2one('res.partner', compute='_compute_partner_id', string="Partner", store=True, readonly=True)
     amount = fields.Monetary(compute='_amount_compute', store=True)
     narration = fields.Text(string='Internal Note')
-    company_id = fields.Many2one('res.company', related='journal_id.company_id', string='Company', store=True, readonly=True,
-        default=lambda self: self.env.user.company_id)
+    company_id = fields.Many2one('res.company', related='journal_id.company_id', string='Company', store=True, readonly=True)
```

Comment:
@qdp-odoo Could you validate this? To me, the default value is not necessary.

Reply:
indeed, moreover the field is readonly... so a default value here is perfectly useless